### ISSUE TITLE
Config Detail Views Hide Bottom of List

### ISSUE THREAD

**Issue by lonn-u** [sjarvela/kloudspeaker#152]
In Users, Groups, and Folders configuration, when clicking on a list item, the bottom of the list is hidden by the detail view.  Would be nice to simply shorten the list so that the detail doesn't cover it.

**Comment by sjarvela** [comment]
I agree, the detail panel sliding from bottom is far from optimal. I've been trying alternative ways, in event logging UI there is a new revision for the admin list, which shows details below the list and at the same time shortens the list.

Only this requires rewriting each admin UI with the new components, this requires some time.
